Platform/ApollolakeBoardPkg/CfgData/CfgData_BootOption.dsc

```diff
@@ -6,34 +6,18 @@
 ##
 
   # !BSF PAGE:{OS}
-  # !BSF SUBT:{CFGHDR_TMPL:OS_CFG_DATA:0:0}
-
   # !HDR HEADER:{OFF}
-  # !HDR EMBED:{OS_CFG_DATA:TAG_700:START}
-
-
-  # !HDR HEADER:{ON}
-  # !HDR EMBED:{OS_CFG_BOOT:OsBootList:START}
-  gCfgData.Rserved                |      * | 0x03 | 0
-
-  gCfgData.OsBootOptionCount      |      * | 0x01 | 5
 
-  gCfgData.OsBootOptions          |      * |   0  | 0
-  # !HDR EMBED:{OS_CFG_BOOT:OsBootList:END}
-  # !HDR HEADER:{OFF}
-
-  # AOS boot from EMMC # |ImgType|flag|DevType|DevNum|HwPart|SwPart|FsType|  image0 name  | sw0 part| sw0 lba|  image1 name  | sw1 part| sw1 lba|  image2 name  | sw2 part| sw2 lba|
+  #                             # |ImgType|flag|DevType|DevNum|HwPart|SwPart|FsType|  image0 name
   # eMMC boot P1
-  # !BSF SUBT:{OS_TMPL:0 :  0    :  0 :   2   :  0   :   0  :    0 :    0 :'iasimage.bin' :       0 :      0 :     0         :     0   :  0     :     0         :     0   :   0    }
+  # !BSF SUBT:{BOOT_OPTION_TMPL:0 :  0    :  0 :   2   :  0   :   0  :    0 :    0 :'iasimage.bin'}
   # eMMC boot P2
-  # !BSF SUBT:{OS_TMPL:1 :  0    :  0 :   2   :  0   :   0  :    0 :    3 :      ''       :       1 :      0 :     0         :     0   :  0     :     0         :     0   :   0    }
+  # !BSF SUBT:{BOOT_OPTION_TMPL:1 :  0    :  0 :   2   :  0   :   0  :    0 :    3 :      ''      }
   # SATA boot
-  # !BSF SUBT:{OS_TMPL:2 :  0    :  0 :   0   :  0   :   0  :    0 :    0 :'iasimage.bin' :       0 :      0 :     0         :     0   :  0     :     0         :     0   :   0    }
+  # !BSF SUBT:{BOOT_OPTION_TMPL:2 :  0    :  0 :   0   :  0   :   0  :    0 :    0 :'iasimage.bin'}
   # USB boot
-  # !BSF SUBT:{OS_TMPL:3 :  0    :  0 :   5   :  0   :   0  :    0 :    0 :'iasimage.bin' :       0 :      0 :     0         :     0   :  0     :     0         :     8   :   0    }
+  # !BSF SUBT:{BOOT_OPTION_TMPL:3 :  0    :  0 :   5   :  0   :   0  :    0 :    0 :'iasimage.bin'}
```

Comment:
In order to let users to use the new config data structure smoothly, how about providing an example that demos how to support Android A/B with trusty support?

Reply:
I don't have example for now.  Maybe we can add this when a real platform needs to support this feature.

Reply:
Tested some populated boot options with Android A/B information.  It does build boot options properly.